bluebottle/members/admin.py

```diff
@@ -325,9 +328,19 @@ def following(self, obj):
     def reset_password(self, obj):
         reset_form_url = reverse('admin:auth_user_password_change', args=(obj.id, ))
         reset_mail_url = reverse('admin:auth_user_password_reset_mail', kwargs={'user_id': obj.id})
-        return format_html("<a href='{}'>{}</a>  | <a href='{}'>{}</a>  ",
-                           reset_form_url, _("Reset password form"),
-                           reset_mail_url, _("Send reset password mail"))
+        properties.set_tenant(connection.tenant)
+
+        # Don't show frontend password reset mail link for closed sites.
+        if properties.CLOSED_SITE:
```

Comment:
This also means that partner users cannot reset passwords anymore. That does not seem desirable to me.

Reply:
I agree, but now it wasn't working for both.
My suggestion is to add another 'send admin pwd reset' mail link.

Reply:
I do not think I understand. On closed platforms the reset password route is exposed, but just not applicable for most users, or am I missing something?